docker/gen-run-image

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+set -e
+
+PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
+ROOT=$(git rev-parse --show-toplevel)
+
+export DIST="xenial"
+export DMD="dmd1"
+export F="production"
+
+cd $ROOT
+BEAVER_DOCKER_Iimg="sociomantictsunami/dhtnode-builder" beaver dlang install
+beaver dlang make pkg
+
+rm -rf docker-tmp
+mkdir docker-tmp
+cp build/$F/pkg/dhtnode-d*.deb docker-tmp/dhtnode.deb
+cp doc/etc/dht.config.ini docker-tmp/config.ini
+
+cat > docker-tmp/Dockerfile <<- file
+FROM ubuntu:xenial
+COPY dhtnode.deb /packages/
+RUN apt update
+RUN mkdir -p /srv/dhtnode/dhtnode-0 && apt install -y /packages/*.deb
+COPY config.ini /srv/dhtnode/dhtnode-0/etc/
```

Comment:
I _think_ the dhtnode requires `log` and `data` as well.

Reply:
But those should be empty, right? Debian package post-install script creates those for each `/srv/dhtnode/dhtnode-*` dir present.

Reply:
Ah ok. Yep, empty is fine.